src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figures {

```

Comment:
This class should not extend `Figures` class. In Java, we use class inheritance to represent the "is-a" relationship between the child and parent classes. In this case, a `Square` is not a `Figure`, rather it is a type of `Figure`. Hence, it would be more appropriate to have `Square` implement a `Figure` interface.